kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -360,9 +360,7 @@ impl<'tcx> GotocCtx<'tcx> {
         args.iter()
             .filter_map(|o| {
```

Comment:
Is `filter_map` still needed? Looks like both arms return `Some`

Reply:
Thanks, fixed.